### ISSUE TITLE
Webpage not showing; Shiny app not working

### ISSUE THREAD

**Issue by bobaekang** [uc-cfss/fp-bobaekang#12]
I have two issues here,

1. My webpage address (https://uc-cfss.github.io/fp-bobaekang/) is not working. However, if I try the addresses to individual html pages, I can reach them. (example: https://uc-cfss.github.io/fp-bobaekang/home.html)

2. The apps I have included in the third page (https://uc-cfss.github.io/fp-bobaekang/result_m.html) fail to show. I keep see the message reading, "Disconnected from the server." The apps work locally. I do not understand how to amend this issue...

Please help me! Thank you.

**Comment by bensoltoff** [comment]
1. You need to save the main page as `index.Rmd` and `index.html`
1. That is something related to the app itself on shinyapps.io. Going directly to the link I cannot access the app. Check your settings on shinyapps.io for that app. Does it give you any error messages there? Have you run out of free computing time for the month? Does the app still run locally on your computer?

**Comment by bobaekang** [comment]
@bensoltoff 
1. Thank you for your advice on the name of the `Rmd` and `html` files. I think the site shows up normally when I use https://uc-cfss.github.io/fp-bobaekang/.
2. I tried the Restart button on the Shiny dashboard page. I wonder if this is because the app is too large? But I do not have much going on here as you may see in my script for either app--they are in fact almost identical to each other except for what data they use. Only thing that may add more weight to this is using ggmap to generate an interactive map object... I will report to you after trying it without the map plot.

**Comment by bobaekang** [comment]
@bensoltoff 

Now that I have removed the map part, the app seems to work just fine. I thought having an interactive map would look nice here, but maybe I have to go without it. I would love some suggestions if you have any. Thank you!

**Comment by bensoltoff** [comment]
It might be related to the Shiny server run by shinyapps.io. A quick google search found [a related problem](https://groups.google.com/forum/#!topic/shiny-discuss/7Y7347HAvEU) but no solution was ever posted. To confirm, the app does work when you run it locally?

**Comment by bobaekang** [comment]
@bensoltoff 
They both *ran* locally, with all their interactivity. Now I have changed the apps to have simpler graphs for the trip duration variable, and they both run without issues (please see [here](https://uc-cfss.github.io/fp-bobaekang/result_m.html)). I have not yet committed the changes in my apps so please take a look at my repo folders (`fp-app1` and `fp-app2`) and the scripts inside and share your thoughts with me. Otherwise, I will commit the changes to apps before noon. Thank you!

**Comment by bensoltoff** [comment]
I could deploy [`fp-app1` without issue](https://bensoltoff.shinyapps.io/fp-app1/). I don't know why it won't work for you. One thing you can try is completely delete the apps from shinyapps.io, then re-deploy them rather than updating the existing app. Not sure if that will do anything or not. Seems like it might be a problem with shinyapps.io, not the app itself.

**Comment by bobaekang** [comment]
@bensoltoff 
I will then try deploying completely new apps using the same scripts. Could you give me the address to your app please? Thank you.

**Comment by bensoltoff** [comment]
See the link in the previous comment

**Comment by bobaekang** [comment]
@bensoltoff 
The same scripts, still not working. I wonder if this has something to do with the instance size limit with my free account? Anyhow, I won't be able to use the interactive maps if I cannot deploy them with my shinyapps account. But it is good to know that my apps have no inherent problem! Thank you.

**Comment by bensoltoff** [comment]
Yeah, that could be an issue. You could pay for one month on shinyapps.io to get a larger instance size limit (I think it's $9). Otherwise go with a non-interactive version

**Comment by bensoltoff** [comment]
Though I just deployed it with the same instance size and it ran fine. Might be running into a total hours limit, though even with the free plan you get 25 hours per month. I really have no clue ☹️
